src/components/CollectionDetailPage/CollectionItem/CollectionItem.tsx

```diff
@@ -137,6 +145,7 @@ export default class CollectionItem extends React.PureComponent<Props> {
                 >
                   <Dropdown.Menu>
                     <Dropdown.Item text={t('collection_item.see_details')} as={Link} to={locations.itemDetail(item.id)} />
+                    <Dropdown.Item text={t('collection_context_menu.see_in_world')} onClick={this.handleNavigateToExplorer} />
```

Comment:
just as a heads up, i think this can be done easier using the `href` prop instead of `onClick`